bench/Evaluation.hs

```diff
@@ -1,9 +1,11 @@
 {-# LANGUAGE DataKinds, FlexibleContexts, PackageImports, PartialTypeSignatures, TypeApplications, TypeFamilies #-}
 
-module Main where
+module Evaluation (benchmarks) where
```

Comment:
The benchmarks had broken; this fixes them up and organizes them a little more fluently. I opted not to include my benchmarks of `catFile`, since `catFile` is internal to semantic.

Reply:
As we just talked about in Slack, definitely in favour of building with `--enable-benchmarks` to avoid this recurring.